src/components/WeeklySummary/WeeklySummary.jsx

```diff
@@ -70,33 +77,40 @@ export class WeeklySummary extends Component {
     const summary = (weeklySummaries && weeklySummaries[0] && weeklySummaries[0].summary) || '';
     const summaryLastWeek =
       (weeklySummaries && weeklySummaries[1] && weeklySummaries[1].summary) || '';
-    const summaryBeforeLast =
+    const summaryTwoWeeksAgo =
       (weeklySummaries && weeklySummaries[2] && weeklySummaries[2].summary) || '';
+    const summaryThreeWeeksAgo =
+      (weeklySummaries && weeklySummaries[3] && weeklySummaries[3].summary) || '';
 
     const dueDateThisWeek = weeklySummaries && weeklySummaries[0] && weeklySummaries[0].dueDate;
     // Make sure server dueDate is not before the localtime dueDate.
     const dueDate = moment(dueDateThisWeek).isBefore(this.state.dueDate)
       ? this.state.dueDate
       : dueDateThisWeek;
     const dueDateLastWeek =
-      (weeklySummaries && weeklySummaries[1] && weeklySummaries[1].dueDate) ||
+      // (weeklySummaries && weeklySummaries[1] && weeklySummaries[1].dueDate) ||
       this.state.dueDateLastWeek;
     const dueDateBeforeLast =
-      (weeklySummaries && weeklySummaries[2] && weeklySummaries[2].dueDate) ||
+      // (weeklySummaries && weeklySummaries[2] && weeklySummaries[2].dueDate) ||
```

Comment:
Please, @LeeSingChanYau, delete these comments. Thereafter, I'm going to approve your PR! Thank you!